packages/main/src/themes/TableRow.css

```diff
@@ -90,4 +90,26 @@
 :host([selected][type="Active"]) .ui5-table-row-root:hover:not(:active),
 :host([selected][mode="SingleSelect"]) .ui5-table-row-root:hover:not(:active) {
 	background-color: var(--sapList_Hover_SelectionBackground);
-}
\ No newline at end of file
+}
+
+/* Navigated */
+:host([navigated]) tr {
+	position: relative;
+}
+:host([navigated]) tr:after {
+    display: block;
+    content: '';
+	width: 100%;
+	position: absolute;
+	inset: 0px;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
```

Comment:
You can leave "inset: 0" as it combines the four metrics into one.

Reply:
I changed it.